change/@fluentui-react-virtualizer-bab73d05-7372-4052-8d22-ca75b5fadfe2.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32375&commitId=8258215d3609ba1cd0ac23bf4f358749963e75a5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|21| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|3302| Changed|
|Drawer.Full Overlay RTL.chromium.png|1141| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>